pkg/cli/common.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/containers/buildah"
 	"github.com/containers/buildah/util"
-	"github.com/opencontainers/runtime-spec/specs-go"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
```

Comment:
just curious why you added this?

Reply:
Ew, I was giving Visual Studio Code a try and some go-plugin must have changed that. I'll revert that and go back to my beloved editor of choice.